scapy/arch/windows/__init__.py

```diff
@@ -92,9 +92,21 @@ def run(self):
                 self.event.set()
             else:
                 self.buffer.append(read_line)
-    def query(self, cmd):
+    def query(self, command, cmd=False):
```

Comment:
Could you add some docstring, please? The code looks ok, but from the maintenance perspective it would just benefit.